src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.NativeRuntime.targets

```diff
@@ -0,0 +1,66 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.NativeRuntime.targets
```

Comment:
We could consider putting the code here in `Microsoft.Android.Sdk.CoreCLR.targets`, to be clear it's specific to CoreCLR.

But I'm fine for it to be its own file.

Reply:
The idea was that it would also serve MonoVM. Not sure if we'll ever get to it now, but I think it's better for it to stay as it is for now.